spec/order_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'order'
+
+describe Order do
+  describe "#initialize" do
+    # I would normally think this test as unecessary, but in this case I want to make sure that
+    # if the input method for the menu changes, it muct still return an array. Is that correct?

```

Comment:
That's correct, but in that case you'd need to inject a mock or double of `Menu` into the order, and then try this test with different cases from what `Menu` data is:
```ruby
menu = Menu.new("test_menu.csv")
order = Order.new(menu)

# expect that it should be an array containing the menu items

menu = Menu.new("wrong_menu.csv")
order = Order.new(menu)

# expect menu should default to an empty array, even if the input CSV data was wrong...
```

but obviously this depends on how you'd like your program to behave in case the input data is wrong — since that's not specified in the instructions, that's up to you!